js_modules/dagster-ui/packages/ui-core/src/assets/AssetNodeOverview.tsx

```diff
@@ -317,17 +317,19 @@ export const AssetNodeOverview = ({
       </AttributeAndValue>
 
       <AttributeAndValue label="Code location">
-        <Box flex={{direction: 'column'}}>
+        <Box flex={{direction: 'column', alignItems: 'flex-start', gap: 8}}>
```

Comment:
This is a drive-by fix for this scenario:

![image](https://github.com/user-attachments/assets/29f293d1-785b-4086-8e9a-3f9bb621d746)


after:

![image](https://github.com/user-attachments/assets/de3a48b6-7aa3-4a8f-90fb-d4f989d2f9cb)

Reply:
I feel like it would be nice to have a mini-alert component, these always feel too big for sidebars.

Reply:
Yeah I agree, there's at least one that appears in the asset graph's sidebar and I think in that scenario we place it at the top with no whitespace around it all. Will see if Josh has thoughts about this in a follow-up